perfkitbenchmarker/benchmark_spec.py

```diff
@@ -182,6 +195,21 @@ def ConstructVirtualMachines(self):
         os_type = self._GetOsTypeForGroup(group_name)
         cloud = self._GetCloudForGroup(group_name)
         providers.LoadProvider(cloud.lower())
+        ProviderInfoClass = provider_info.GetProviderInfoClass(cloud.lower())
+
+        benchmarkOkay = True
```

Comment:
Also ConstructVirtualMachines is already on the long side - to help with readability would you mind adding a helper and calling that instead of doing all the checking right here?